### PR TITLE
Clean-Up JenkinsfileBase

### PR DESCRIPTION
**Pull request by nikolamilijevic1**
Removed variable iterationCount,  simply replaced by the counter "i" in the for-loop.

Related to: #1728 
Signed-off-by: <nmilijev@uwaterloo.ca> Nikola Milijevic

### CODE REVIEW

**Comment by nikolamilijevic1** [comment]
@llxia @renfeiw

**Comment by llxia** [comment]
Please remove `iterationCount++`

**Comment by nikolamilijevic1** [comment]
> Please remove `iterationCount++` and provide Grinder link.

Removed and https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/10374/

**Comment by renfeiw** [comment]
Please remove the internal grinder link and provide the the build number only.

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -546,7 +545,7 @@ def runTest( ) {
 			}
 			RUNTEST_CMD = "./openjdk-tests ${TARGET} ${CUSTOM_OPTION}"
 			for (int i = 0; i < ITERATIONS; i++) {
```
Comment (renfeiw) [inline buildenv/jenkins/JenkinsfileBase:547]: use `for (int i = 1; i <= ITERATIONS; i++)` instead of `${i+1}`

**Comment by nikolamilijevic1** [comment]
Updated grinder @renfeiw 
| BUILD | PLATFORM | TARGET  | BUILD_LIST | JAVA_VERSION |
--- | --- | --- |  --- |--- |
| Grinder/10406| x86-64_linux| floatSanityTests| functional | 11 |

**Review by llxia — APPROVED** [review]
LGTM

**Review by renfeiw — APPROVED** [review]

**Pull request merged**